packages/editor-ui/src/components/helpers.ts

```diff
@@ -1,5 +1,21 @@
+import dateformat from 'dateformat';
+
+const KEYWORDS_TO_FILTER = ['API', 'OAuth1', 'OAuth2'];
+
 export function addTargetBlank(html: string) {
 	return html.includes('href=')
 		? html.replace(/href=/g, 'target="_blank" href=')
 		: html;
 }
+
+export function convertToDisplayDate (epochTime: number) {
+	return dateformat(epochTime, 'yyyy-mm-dd HH:MM:ss');
+}
+
+export function convertToHumanReadableDate (epochTime: number) {
+	return dateformat(epochTime, 'd mmmm, yyyy @ HH:MM Z');
```

Comment:
Unlike American English style, I think we do not use a comma between month and year when the date starts with the day.

Reply:
will discuss with product